capstone-app/src/app/modules/maps/map/map.component.ts

```diff
@@ -1,13 +1,80 @@
 /// <reference types="googlemaps" />
-import { Component } from '@angular/core';
+import { Component,  ViewChild, ElementRef } from '@angular/core';
+import { MapInfoWindow, MapMarker } from '@angular/google-maps';
 
 @Component({
   selector: 'app-map',
   templateUrl: './map.component.html',
   styleUrls: ['./map.component.scss']
 })
+
 export class MapComponent {
-  // these coordinates are for a location in Miami
-  location = new google.maps.LatLng(26.011760, -80.139050);
+  @ViewChild('searchBar') searchBar: ElementRef;
+  @ViewChild(MapInfoWindow) infoWindow: MapInfoWindow;
+
+  location!: google.maps.LatLng;
```

Comment:
Since location is not guaranteed to be set, I recommend using a ? instead of !

```
location?: google.maps.LatLng
```

This will actually make it a union type with undefined, so you may have to add more code checking that location is set, but it will prevent bugs in the future.